enderio-base/src/main/java/com/enderio/base/EnderIO.java

```diff
@@ -33,13 +33,17 @@
 import javax.annotation.Nonnull;
 import java.io.IOException;
 import java.nio.file.Files;
+import java.util.logging.LogManager;
+import java.util.logging.Logger;
 
 @Mod(EnderIO.MODID)
 public class EnderIO {
     public static final @Nonnull String MODID = "enderio";
 
     private static final Lazy<Registrate> REGISTRATE = Lazy.of(() -> Registrate.create(MODID));
 
+    public static final Logger LOGGER = LogManager.getLogManager().getLogger(MODID);
```

Comment:
Just as a note for future - might be worth utilising logging more now that I think about it.